### PR TITLE
import remaining shared reports

### PR DESCRIPTION
**Pull request by ericcf**
[#88546774]

### CODE REVIEW

**Comment by ericcf** [comment]
Related to: https://github.com/cbitstech/think_feel_do_dashboard/pull/10

File: spec/models/ability_spec.rb [hunk 1]
```diff
@@ -44,14 +44,17 @@ def role(name)
...
+      end
     end
 
     describe "Non-superuser, multi-role Users" do
       it "share all Roles' abilities" do
         expect(multi_role.can?(:manage, BitCore::ContentModule)).to eq true
         expect(multi_role.can?(:manage, Task)).to eq true
-        expect(multi_role.can?(:manage, ThinkFeelDoDashboard::Reports::Comment))
-          .to eq true
+        expect(multi_role.can?(:read, "Reports")).to eq true
```
Comment (gwensmu) [inline spec/models/ability_spec.rb]: Does it make sense for authorization specs to test if a user who shouldn't be able to read reports can't? I don't see that pattern in the overall spec, is that covered elsewhere?

Comment (ericcf) [inline spec/models/ability_spec.rb]: Yes it does. No I don't think it's covered.


--
Eric Carty-Fickes


-------- Original message --------

**Comment by gwensmu** [comment]
Cool, I'd like to see it here at least for this spec.

**Comment by ericcf** [comment]
@michaelwehrley yes: https://github.com/cbitstech/think_feel_do_dashboard/pull/10/files#diff-df86902ce9dceb01164aa1b2f4e97aed

**Pull request merged**